### PR TITLE
fix: expired token in test project

### PR DESCRIPTION
**Pull request by msimionati**
Was detected that an expired token was being used in the tests and the tests started to fail
The token was updated with a valid one.

### CODE REVIEW

File: Doppler.AccountPlans.Test/GetCalculateUpgradeCostTestTest.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ namespace Doppler.AccountPlans
     public class GetCalculateUpgradeCostTest : IClassFixture<WebApplicationFactory<Startup>>
     {
         private const string TokenAccount123Test1AtTestDotComExpire20330518 =
-            "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc1NVIjp0cnVlLCJpYXQiOjE1ODQ0NjM1MjksImV4cCI6MTY0NTU2NzUyOX0.eJZYYKZOw439rWO98urRkmR4H9UqhVoD870RECWtOQlsUEpTeNPFgQfz-bQHyYZ73lkNunvmW4s31wJB1BFrVr4JqklhvM0zEem70aGDrxa1WwNB268p-dzGZqG_RAU4k7oW1JwSvrg6DGVQTlP2DyeNvyO1qwpt8m0yS4z6x3jTzn4V-6iEOIBMp2DxwzMU6x03e_I4jZJRal2T9941BoM88c2S-IXWR_uS0AGbH1Yl7ZMUSx7wQEZi2o7UpOefR6TpKkOTj5iSeSMSVvpGvOjKP5Lk_Y7TSLpNEKTcMaahA1DK4R_EYAbtiIWIvSscEqxlWBzo7AB1MQdNbqrVlA";
+            "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc1NVIjp0cnVlLCJleHAiOjIwMDAwMDAwMDB9.rUtvRqMxrnQzVHDuAjgWa2GJAJwZ-wpaxqdjwP7gmVa7XJ1pEmvdTMBdirKL5BJIE7j2_hsMvEOKUKVjWUY-IE0e0u7c82TH0l_4zsIztRyHMKtt9QE9rBRQnJf8dcT5PnLiWkV_qEkpiIKQ-wcMZ1m7vQJ0auEPZyyFBKmU2caxkZZOZ8Kw_1dx-7lGUdOsUYad-1Rt-iuETGAFijQrWggcm3kV_KmVe8utznshv2bAdLJWydbsAUEfNof0kZK5Wu9A80DJd3CRiNk8mWjQxF_qPOrGCANOIYofhB13yuYi48_8zVPYku-llDQjF77BmQIIIMrCXs8IMT3Lksdxuw";
```
Comment (andresmoschini) [inline Doppler.AccountPlans.Test/GetCalculateUpgradeCostTestTest.cs:24]: This new one is a token of the account `123`, with the email `test1@test.com` and, expires on _May 18th, 2033_?

Comment (msimionati) [reply Doppler.AccountPlans.Test/GetCalculateUpgradeCostTestTest.cs:24]: the previous one was not tied to an account, it only had: "isSU", "iat" and  "exp", the new one has "IsSU" and "exp" only... and the expiration date is the one you indicate in the commentary, 05/18/2033

Comment (andresmoschini) [reply Doppler.AccountPlans.Test/GetCalculateUpgradeCostTestTest.cs:24]: Mmm, in that case, what do you think about renaming the constant (`TokenAccount123Test1AtTestDotComExpire20330518`)? 

I know that it is out of the scope of your PR, but it was wrong in the first place.

Comment (msimionati) [reply Doppler.AccountPlans.Test/GetCalculateUpgradeCostTestTest.cs:24]: yes, it was wrong at the base, I change it

**Comment by jhoffman-ms** [comment]
👍

**Pull request merged**